src/main/java/io/github/gitbucket/markedj/Lexer.java

```diff
@@ -170,6 +171,12 @@ protected void token(String src, boolean top, boolean bq, LexerContext context){
                 }
             }
 
+			{
+				for (Extension extension : options.getExtensions()) {
+					src = extension.lex(src, context, this::token);
```

Comment:
Should continue if any rule is applied?

Reply:
The interface needed to be expanded here, but I think it should fit now.